pkg/minikube/assets/vm_assets.go

```diff
@@ -139,24 +139,31 @@ func (m *MemoryAsset) Read(p []byte) (int, error) {
 }
 
 func CopyFileLocal(f CopyableFile) error {
-	os.MkdirAll(f.GetTargetDir(), os.ModePerm)
+	err := os.MkdirAll(f.GetTargetDir(), os.ModePerm)
+	if err != nil {
+		return errors.Wrapf(err, "error making dirs for %s", f.GetTargetDir())
+	}
 	targetPath := filepath.Join(f.GetTargetDir(), f.GetTargetName())
-	os.Remove(targetPath)
+	err = os.Remove(targetPath)
```

Comment:
I will only remove the file if it exists, I made the opposite choice last time but will do this explicitly to have more verbose errors.